docs/src/format.md

```diff
@@ -0,0 +1,400 @@
+- `operationType` [STRING, REQUIRED]:
+  Specifies the type of operation to be performed on the target. Repeating parts of the PEtab standard, allowed values are:
+  - `setValue`: Sets the current value of the target to the value specified in `targetValue`. The target must be a constant target.
+  - `setRate`: Sets the time derivative of the target to `targetValue`. The target must be a differential target.
+  - `setAssignment`: Sets the target to the symbolic value of `targetValue`. The target must be an algebraic target.
+  - `setParameter`: Sets the value of a model parameter to a dynamic neural network output. `targetValue` must be a neural network output.
+
+- `targetValue` [STRING, REQUIRED]:
+  The value or expression that will be used to change the target. The interpretation of this value depends on the specified `operationType`.
+
+### The `operationType`, mapping table, and condition table defines hybrid model type
+
+The `operationType`, mapping table, and condition table together specify where a neural network model is integrated into a PEtab SciML problem. In particular (here, we refer to neural network inputs and outputs as simply inputs and outputs):
+
+1. **Neural network models in the ODE model’s right-hand side (RHS)**: If all inputs use `setAssignment` and all outputs use either `setRate` or `setParameter`, the neural network appears in the ODE right-hand side.
+2. **Neural network models in the observable function**: If all inputs use `setAssignment` and no outputs are used in the hybridization table, then the outputs must be used in the observable formulas.
+3. **Neural network models to parametrize ODEs**: If all inputs are constant targets, that is use `setValue`, are assigned in the condition table, or are assigned to a constant variable in the mapping table, and if all outputs assign using `setValue` or are assigning in the condition table, then the neural network sets model parameters or initial values before the simulation.
+
+All other combinations are disallowed because they generally do not make sense in a PEtab context. An important consequence of this is that if all inputs are assigned to constant targets, then neural network outputs must also be assigning to constant targets in either the hybridization or condition tables. The `petab_sciml` library provides a linter to ensure that no disallowed combination is used. For packages that do not wish to depend on Python, it is strongly recommended to verify that the input combinations in the condition table are valid.
+
+!!! note "Model structure altering assignments"
+    When `setRate`, `setAssignment`, or `setParameter` are used, the model structure or observable formula is altered during model import. Effectively, a neural network is inserted into the generated functions. PEtab SciML only supports such alterations if they are applied to all conditions; hence, these alterations can only be specified in the hybridization table.

```

Comment:
Would drop this note. What information do we provide here that influences user behaviour?
- we no longer have operators in the condition table, so this statement will be more confusing than helpful
- if a user actually wants condition specific models, we don't really tell him how (encode multiple alternatives through parametrization and assign in condition table).